contributing.md

```diff
@@ -0,0 +1,257 @@
+
+Don't forgot the tests if you are creating a feature, and check twice the orthography if you is creating a documentation ou fixing that
+
+We have an community chimes in with helpful advice if you have questions.
+
+* [Telegram Community - NodeSchool Campinas](https://t.me/nodeschool_campinas)
+
+## To-do before finishing a pull request
+
+Bellow you can get some informations to keep your code acceptable in the pull request, be attempt to follow the guideline at maximum to not being rejected or need to make changes, remember to make changes concized in the theme of issue not try to do something unnecessary. Keep changes easy for others to understand and, in the case of a change request of your pull request, keep an eye out, if you keep in hold too long time without response the pull the request can be canceled by the moderator and reatribuited to another person.
+
+### Commands to check before finishing a pull request!
+
+Every command of the CLI inside the project must be operative and the tests before your changes cannot fail on commit, if you implemented a new feature and this feature increments or decrements the operations, you need to update the test file and the [readme.md](./readme.md) or open an [issue](https://github.com/victorperin/qr-scanner-cli/issues/new) for changing the documentation with a great description for what to do and referencing the feature.
+
+### Husky the pre-commit flow
+
+Husky is an hook that operates before commit changes in the project the hook lint the commit message with commitlint to follow a guideline, on the development of this project you need to follow some those to can commit your project, the commit message has 
+
+#### Remember to follow it, otherwise your code will be rejected
+
+The commit message need keep a standard, so we chose to use the commitlint with angular configuration, the fully documentation can be [find here](https://www.npmjs.com/package/@commitlint/config-angular), but the abstract of the documentation will be bottom:
+
+The commitmessage need to start with a type, the acceptable type's is:
+```javascript
+types => ['build','ci','docs','feat','fix','perf','refactor','revert','style','test']
+```
+
+And the type-case is ruled by lowercase always if is in uppercase or camelcase is not valid.
+
+After the type is optional to describe the scope of changes also in lowercase to be valid and if is used need to be inside pharentesis other case if is not used need to be nothing has changed.
+
+And ```:``` with the message of commit objective; the full commit message wold'nt finish with ```.``` and can't pass to 72 chars or will be disconsidered by hook.
+
+> To keep pretty, please dedicate an space before the message commit
+
+These are some cases to exemplify the commit message:
+
+**Commit message**|**Accpetable**
+-----|-----
+TYPE:commit message|:-1:
```

Comment:
I think this :+1: :-1: emojis are too much alike. If we change them, could become easer to read.